pyguide.md

```diff
@@ -2320,23 +2322,13 @@ def my_method(self, first_var: int) -> int:
 ```
 
 If the combination of the function name, the last parameter, and the return type
-is too long, indent by 4 in a new line.
-
-```python
-def my_method(
-    self, first_var: int) -> Tuple[MyLongType1, MyLongType1]:
-  ...
-```
-
-When the return type does not fit on the same line as the last parameter, the
-preferred way is to indent the parameters by 4 on a new line and align the
-closing parenthesis with the def.
+is too long, indent by 8 in a new line.
 
 ```python
-Yes:
 def my_method(
-    self, **kw_args: Optional[MyLongType]
-) -> Dict[OtherLongType, MyLongType]:
```

Comment:
I'd say let's make an exception for `self` and allow two paramaters in one row in cases like this. The argument with changing paramater's names doesn't apply here, because `self` never changes.

Reply:
Unless you change it to class or static method.
"cases like this" meaning two params method? I'm not in favor but don't care that much.

Reply:
By "cases like this" I meant "two parameters, one of the being `self`".